packages/emitter-framework/src/python/components/constant-declaration.tsx

```diff
@@ -0,0 +1,22 @@
+export interface ConstantDeclarationModel {
+  name: string;
+  value: string | number | undefined;
+}
+
+export function ConstantDeclaration({ name, value }: ConstantDeclarationModel) {
+  // COMMENT: Can't use `useNamePolicy` here because I can't query strings directly.
```

Comment:
I'm not sure I understand the limitation here, can you help me understand it with an example?

Reply:
This was true in the pre-alloy version of name policy. It's no longer the case.